outputs/policyadapter.go

```diff
@@ -0,0 +1,60 @@
+package outputs
+
+import (
+	"fmt"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	//"log"
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/falcosecurity/falcosidekick/types"
+	"k8s.io/client-go/tools/clientcmd"
+	// "github.com/google/uuid"
+)
+
+func NewPolicyReportClient(config *types.Configuration, stats *types.Statistics, promStats *types.PromStatistics, statsdClient, dogstatsdClient *statsd.Client) (*Client, error) {
+
+	if config.PolicyReport.Kubeconfig != "" {
+		restConfig, err := clientcmd.BuildConfigFromFlags("", config.PolicyReport.Kubeconfig)
+		if err != nil {
+			return nil, err
+		}
+		clientset, err := kubernetes.NewForConfig(restConfig)
+		if err != nil {
+			return nil, err
+		}
+		return &Client{
+			OutputType:       "PolicyReport",
+			Config:           config,
+			Stats:            stats,
+			PromStats:        promStats,
+			StatsdClient:     statsdClient,
+			DogstatsdClient:  dogstatsdClient,
+			KubernetesClient: clientset,
+		}, nil
+	}
+	restConfig, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, fmt.Errorf("unable to load in-cluster config: %v", err)
+	}
+	clientset, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		return nil, err
+	}
+	return &Client{
+		OutputType:       "PolicyReport",
+		Config:           config,
+		Stats:            stats,
+		PromStats:        promStats,
+		StatsdClient:     statsdClient,
+		DogstatsdClient:  dogstatsdClient,
+		KubernetesClient: clientset,
+	}, nil
+
+}
+
+// PolicyAdapterPost receives falco payload
```

Comment:
```suggestion
// PolicyReportPost creates Policy Report Resource in Kubernetes
```

I think, `PolicyReportCreate` might be a better name, more sense, wdyt?

Reply:
Yes, definitely. Will make this correction.